standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -3342,8 +3342,8 @@ public List<String> listPartitionNames(final String catName, final String dbName
     final boolean isEmptyFilter = exprBytes.length == 1 && exprBytes[0] == -1;
     ExpressionTree tmp = null;
```

Comment:
I think we can remove the **ExpressionTree tmp = null;**  as it is not used anymore as you have removed it in the if condition.